Combine 'apps' With 'inspect'

WestleyK opened the pull request:
## Description of the Pull Request (PR):

This PR will combine `apps` with `inspect`.

## This fixes or addresses the following GitHub issues:

- Fixes #3344 

<br>

Commit:
Merged apps with inspect

Commit:
Merge remote-tracking branch 'upstream/master' into app-inspect

Commit:
Removed extra file

Commit:
FINALLY working!

Commit:
Removed extra code

Commit:
Merge remote-tracking branch 'upstream/master' into app-inspect

mem commented on cmd/internal/cli/inspect_linux.go:
```diff
@@ -27,6 +27,8 @@ import (
 	singularityConfig "github.com/sylabs/singularity/pkg/runtime/engines/singularity/config"
 )
 
+const listAppsCommand = "echo apps:`ls \"$app/scif/apps\" | wc -c`; for app in ${SINGULARITY_MOUNTPOINT}/scif/apps/*; do\n    if [ -d \"$app/scif\" ]; then\n        APPNAME=`basename $app`\n        echo \"$APPNAME\"\n    fi\ndone\n"
```
Please add quotes around `$app` when setting APPNAME.
```suggestion
const listAppsCommand = "echo apps:`ls \"$app/scif/apps\" | wc -c`; for app in ${SINGULARITY_MOUNTPOINT}/scif/apps/*; do\n    if [ -d \"$app/scif\" ]; then\n        APPNAME=`basename \"$app\"`\n        echo \"$APPNAME\"\n    fi\ndone\n"
```

mem committed:
Update cmd/internal/cli/inspect_linux.go

Trying out suggesting mode commit. If something got screwed up, blame -> @WestleyK!

Co-Authored-By: WestleyK <westley@sylabs.io>

Commit:
Merge remote-tracking branch 'upstream/master' into app-inspect

ikaneshiro commented on cmd/internal/cli/inspect_linux.go:
```diff
@@ -178,6 +186,11 @@ var InspectCmd = &cobra.Command{
 		a := []string{"/bin/sh", "-c", ""}
 
+		if apps {
+			sylog.Debugf("Inspecting apps in container")
```
Can we change this to `Inspection of apps selected.` So that all the debug statements are really similar?

ikaneshiro commented on cmd/internal/cli/inspect_linux.go:
```diff
@@ -260,6 +273,10 @@ var InspectCmd = &cobra.Command{
 			fmt.Println(string(jsonObj))
 		} else {
+			if inspectObj.Attributes.Apps != "" {
+				fmt.Printf("==Apps==\n")
```
Lets make this `==apps==` for the same reason as above

Commit:
Merge remote-tracking branch 'upstream/master' into app-inspect

Commit:
Fixed panic

Commit:
Updated debug message

ikaneshiro reviewed (approved):
lgtm
